components/docs-chef-io/content/automate/servicenow_event_creation.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "ServiceNow Event Creation App"
+
+draft = false
+
+gh_repo = "automate"
+
+[menu]
+  [menu.automate]
+    title = "Event Creation App"
+    identifier = "automate/integrations/servicenow/servicenow_event_creation.md Chef Automate Event Creation"
+    parent = "automate/integrations/servicenow"
+    weight = 20
++++
+
+The Chef Automate Event Creation App is a ServiceNow-certified app available in the [ServiceNow](https://store.servicenow.com) store. The Event Creation App integrates Chef Automate with ServiceNow by enabling you to create events and alerts corresponding to the failed Chef Infra client runs and compliance scans. These events and alerts generated in the ServiceNow platforms are due to the failures in the Chef Automate infrastructure automation updates and automated compliance checks. Thus, this app captures the failures in your automated infrastructure and improves your event tracking and resolution.
+
+The Event Creation App exposes the REST API endpoint for communication between Chef Automate and the ServiceNow instance. Chef Automate sends HTTPS JSON notifications to the Event Creation App in a ServiceNow instance to create and update events.
+
+## User Requirements
+
+- Your unique ServiceNow URL. It has the format: `https://ven12345.service-now.com`.
+- Setting up the Event Creation App requires the `x_chef_automate.api` role in configuring it. Your ServiceNow administrator can enable this for you.
+
+## System Requirements
+
+- The [Integration App]({{< relref "servicenow_integration" >}}) is already installed and configured.
+
+## Required ServiceNow Plugins
+
+Install the `Event management Core` plugin from the Service Management dashboard. Kindly contact the ServiceNow administrator for further details.
+
+## Install
+
+Get the Chef Automate Event Creation App from the [ServiceNow](https://store.servicenow.com) store and then install it to your account from the **Service Management** dashboard.
+
+! the app is not yet releases and hence we cannot find in the store
+
+## Setup
+
+You can set up automatic event creation and alerts for:
+
+- Chef Infra client run alerts
+  Includes the client run alerts grouped by the message key. The message key is either set as cookbook or node in the properties section.
+
+- Chef InSpec scan alerts
```

Comment:
The formatting on all of these bullet points needs fixed.
![Screen Shot 2021-12-09 at 12 21 20 PM](https://user-images.githubusercontent.com/4400151/145469835-ac836786-53ff-43b8-a35c-c444159ff990.png)